rebar.config

```diff
@@ -6,7 +6,9 @@
     {ranch, {git, "https://github.com/ninenines/ranch.git", {tag, "1.2.1"}}},
     {eper, ".*", {git, "https://github.com/massemanet/eper.git", {tag, "0.97.1"}}},
     {eflame, ".*", {git, "https://github.com/slfritchie/eflame.git", master}},
-    {amqp_client, ".*", {git, "https://github.com/jbrisbin/amqp_client.git", master}}
+    {amqp_client, ".*", {git, "https://github.com/jbrisbin/amqp_client.git", master}},
```

Comment:
You could find it in [this](https://github.com/dcos/spartan/pull/41/files#diff-31d7a50c99c265ca2793c20961b60979R9) PR.